bitnami/kafka/templates/_helpers.tpl

```diff
@@ -157,6 +157,23 @@ Return true if encryption via TLS for client connections should be configured
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return the configured value for the external client protocol, defaults to the same value as clientProtocol
```

Comment:
This isn't true: it's returning a boolean. If you want to return the 1st non-empty value, use coalesce:
```
   {{- coalesce .Values.auth.externalClientProtocol .Values.auth.clientProtocol -}}
```

Reply:
@juan131 As I understand it `or` is the same or as in go-templates and it's returning the first value that evaluates as boolean true.  See: https://pkg.go.dev/text/template#hdr-Functions. I opted for this because I wanted it to work with empty strings as well as undefined/null values. But coalesce might be better if it works with empty strings?

Reply:
It might work using `or` because of this particular implementations of `or` in go-templates. That said, I find it confusing (so I guess it can be confusing for others too). I'd stick to the `coalesce` function for consistency with the res of the catalog, see:
- https://masterminds.github.io/sprig/defaults.html#coalesce

Reply:
Ah, didn't realise that coalesce worked with "empty". I'll change it!